### DISCUSSION TITLE
Superset consuming more time

### DISCUSSION THREAD

**Discussion by harsha23339** [apache/superset#22733] [Q&A / Help]
we have a select query which is retrieving 100K rows in about 12 sec's in SSMS.
When we run the same query in superset it is almost taking 50 Sec's.
If multiple users are running, it is timing out after 15mins for few users
Any suggestions on how to reduce the timings on superset, as for even one user superset is almost consuming 4 times the time consumed by SSMS.
Happy to provide any details such as config, query etc.
![image](https://user-images.githubusercontent.com/105300318/186528321-3ed1c058-5fd2-4a26-a4d0-7287194be27d.jpeg)

**Comment by rusackas** [comment]
Sounds like you may not having caching configured correctly. Please see https://superset.apache.org/docs/installation/cache/

Moving this o a Q&A / Help discussion as it sounds like it represents a configuration issue more than a bug, per se.
